src/django_upgrade/fixers/testcase_methods.py

```diff
@@ -0,0 +1,43 @@
+"""
+Replace spelling of assertFormsetError and assertQuerysetEqual:
+https://docs.djangoproject.com/en/4.2/releases/4.2/#miscellaneous
+"""
+from __future__ import annotations
+
+import ast
+from functools import partial
+from typing import Iterable
+
+from tokenize_rt import Offset
+
+from django_upgrade.ast import ast_start_offset
+from django_upgrade.data import Fixer, State, TokenFunc
+from django_upgrade.tokens import find_and_replace_name
+
+fixer = Fixer(
+    __name__,
+    min_version=(4, 2),
+)
+
+MODULE = "django.test.testcase"
+NAMES = {
+    "assertFormsetError": "assertFormSetError",
+    "assertQuerysetEqual": "assertQuerySetEqual",
+}
+
+
+@fixer.register(ast.Call)
+def visit_Call(
+    state: State,
+    node: ast.Call,
+    parents: list[ast.AST],
+) -> Iterable[tuple[Offset, TokenFunc]]:
+    if (
+        isinstance(func := node.func, ast.Attribute)
```

Comment:
we should call `looks_like_test_file` as well

Reply:
like in `assert_form_error`, that is.

It keeps django-upgrade fast by not performing checks in irrelevant files